packages/devtools_app/lib/src/screens/inspector_v2/widget_properties/properties_view.dart

```diff
@@ -0,0 +1,166 @@
+          child: ListView.builder(
+            itemCount: properties.length,
+            itemBuilder: (context, index) {
+              return PropertyItem(
+                index: index,
+                properties: properties,
+              );
+            },
+          ),
+        );
+      },
+    );
+  }
+}
+
+class PropertyItem extends StatelessWidget {
+  const PropertyItem({
+    super.key,
+    required this.properties,
+    required this.index,
+  });
+
+  final List<RemoteDiagnosticsNode> properties;
+  final int index;
+
+  @override
+  Widget build(BuildContext context) {
+    final property = properties[index];
+    final theme = Theme.of(context);
+
+    return DecoratedBox(
+      decoration: BoxDecoration(
+        borderRadius: _calculateBorderRadiusForRow(
+          rowIndex: index,
+          totalRows: properties.length,
+        ),
+        border: _calculateBorderForRow(
+          rowIndex: index,
+          theme: theme,
+        ),
+        color: alternatingColorForIndex(
+          index,
+          Theme.of(context).colorScheme,
+        ),
+      ),
+      child: Row(
+        children: [
+          Expanded(child: PropertyName(property: property)),
+          Expanded(flex: 2, child: PropertyValue(property: property)),
+        ],
+      ),
+    );
+  }
+
+  BorderRadius _calculateBorderRadiusForRow({
+    required int rowIndex,
+    required int totalRows,
+  }) {
+    // Rounded top corners for the first row:
+    if (rowIndex == 0) {
+      return const BorderRadius.only(
+        topLeft: defaultRadius,
+        topRight: defaultRadius,
+      );
+    }
+    // Rounded bottom corners for the last row:
+    if (rowIndex == (totalRows - 1)) {
+      return const BorderRadius.only(
+        bottomLeft: defaultRadius,
+        bottomRight: defaultRadius,
+      );
+    }
+    return BorderRadius.zero;
+  }
+
+  Border _calculateBorderForRow({
+    required int rowIndex,
+    required ThemeData theme,
+  }) {
+    final borderColor = theme.focusColor;
+    return Border(
+      // This prevents the top and bottom borders from being painted next to
+      // to each other, making the border look thicker than it should be:
+      top: rowIndex == 0
+          ? BorderSide(
+              color: borderColor,
+            )
+          : BorderSide.none,
+      bottom: BorderSide(color: borderColor),
+      right: BorderSide(color: borderColor),
+      left: BorderSide(color: borderColor),
+    );
+  }
+}
+
+class PropertyName extends StatelessWidget {
+  const PropertyName({
+    super.key,
+    required this.property,
+  });
+
+  final RemoteDiagnosticsNode property;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+
+    return Padding(
+      padding: const EdgeInsets.all(denseRowSpacing),
+      child: Text(
+        property.name ?? '',
+        style: DiagnosticsTextStyles.regular(
+          theme.colorScheme,
+        ).merge(theme.subtleTextStyle),
```

Comment:
do we still need to be using DiagnosticsTextStyles or can we just switch to using subtleTextStyle ? same question below

Reply:
Good point, we don't!